typescript/package.json

```diff
@@ -1,4 +1,32 @@
 {
+  "name": "kurtosis-js-lib",
+  "version": "1.0.35",
+  "description": "This repo contains clients for communicating with the Kurtosis API server, which both testsuites and Kurtosis Modules will use to manipulate the test environment.",
+  "main": "./build/index",
```

Comment:
Recommend making a specific subdirectory, `build/typescript`, so that you don't inadvertently get cross-language contamination in the future from other languages who dump stuff into `build`